README.md

```diff
@@ -1,107 +1,68 @@
-<div align="center">
-  <h1 align="center">
-    <img src="./www/static/img/logo-text.png" alt="tRPC" height="100" />
-  </h1>
-  <p>End-to-end typesafe APIs made easy</p>
-  <p>
-    <a href="https://codecov.io/gh/trpc/trpc">
-      <img src="https://codecov.io/gh/trpc/trpc/branch/main/graph/badge.svg?token=KPPS918B0G" alt="codecov">
-    </a>
-  </p>
-  <p>
-    <figure>
-      <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Server/client example" />
-      <figcaption>
-        The client above is <strong>not</strong> importing any code from the server, only it's type declarations.
-        <br/>
-        <sub><sup><em><a href="https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export">Import type only imports declarations to be used for type annotations and declarations. It always gets fully erased, so there’s no remnant of it at runtime.</a></em></sup></sub>
-      </figcaption>
-    </figure>
-  </p>
-</div>
-
-
-> 💬 Join the tRPC Discord server to chat to other people using tRPC - [trpc.io/discord](https://trpc.io/discord)
-
-## Intro
```

Comment:
Personally I think this section was already nice bad a good length for describing what tRPC is. All the features listed are important and unique so I don't think they should be compressed.